src/styles/_catalog.scss

```diff
@@ -1,22 +1,22 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  justify-content: center;
+  justify-items: center;
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
```

Comment:
Pay attention to the readme, you have 4 cards since the width of the screen is 944px, but the assignment says it should start on 1024px, and so on